doc/content/gateways/lora-basics-station/_index.md

```diff
@@ -4,19 +4,32 @@ description: ""
 weight: -1
 ---
 
-The [{{% lbs %}}](https://lora-developers.semtech.com/resources/tools/basic-station/welcome-basic-station/) protocol simplifies management of large scale LoRaWAN networks. {{% lbs %}} is the preferred way of connecting Gateways to {{% tts %}}. 
+The [{{% lbs %}}](https://lora-developers.semtech.com/resources/tools/basic-station/welcome-basic-station/) protocol simplifies management of large scale LoRaWAN networks. {{% lbs %}} is the preferred way of connecting Gateways to {{% tts %}}.
+
+This section contains information for connecting your gateway to {{% tts %}} using {{% lbs %}} and its sub protocols.
 
 <!--more-->
 
+## Advantages of {{% lbs %}}
+
 Some of the advantages of {{% lbs %}} over the legacy UDP Packet Forwarder are:
 
 - Centralized Update and Configuration Management
 - TLS and Token-based Authentication
 - Centralized Channel-Plan Management
 - No Dependency on Local Time Keeping
 
+## LNS and CUPS Sub Protocols
+
 {{% lbs %}} contains two sub protocols for connecting Gateways to Network Servers, LoRaWAN Network Server (LNS) and Configuration and Update Server (CUPS).
 
-{{% tts %}} has full support for the {{% lbs %}} protocol. This section contains information for connecting your gateway using LNS or CUPS.
+
+### LoRaWAN Network Server (LNS)
+
+LNS establishes a data connection between a {{% lbs %}} gateway and a Network Server (in this case, {{% tts %}}). LoRa® uplink and downlink frames are exchanged through this data connection. The LNS protocol is **required** for sending and receiving LoRaWAN data.
```

Comment:
I think those are LoRaWAN uplink/downlinks and not LoRa. Good to confirm that.

Reply:
stole it from here https://doc.sm.tc/station/tcproto.html
I guess LoRaWAN uplinks are a layer on top of LoRa in any case